setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
 name = vault-cli
 description = CLI tool for hashicorp vault
-version = 0.4.1
+version = 0.5.1.dev0
 author = ylachiver
```

Comment:
It's time to remove this line I think :)

Reply:
We'll do it someday :+1: